build-system/pr-check.js

```diff
@@ -37,12 +37,11 @@ function timedExecOrDie(cmd) {
 function main() {
   // TODO(danielrozenberg): conditional test running
   timedExecOrDie('eslint .');
-  timedExecOrDie('cd bundle-size && npm ci');
-  timedExecOrDie('cd bundle-size && npm test -u');
-  timedExecOrDie('cd test-status && npm ci');
-  timedExecOrDie('cd test-status && npm test -u');
-  timedExecOrDie('cd pr-deploy && npm ci');
-  timedExecOrDie('cd pr-deploy && npm test -u');
+  ['bundle-size', 'test-status', 'pr-deploy', 'owners'].forEach(appName => {
+    timedExecOrDie(`cd ${appName} && npm ci`);
+    timedExecOrDie(`cd ${appName} && npm test -u`);
```

Comment:
Can we make the owners test less verbose?

Reply:
The verbosity comes from the Probot tests in `index.test.js`, and I'm not really clear on why they show up at all. @erwinmombay do you remember why this happens for those tests? Or how to disable them?

Reply:
You can set different `LOG_LEVELS` or `verbose` levels. Examples here -
https://travis-ci.org/ampproject/amp-github-apps/builds/587212310#L708
https://travis-ci.org/ampproject/amp-github-apps/builds/587212310#L771

Reply:
I have tried that. They are ignored.

Reply:
:man_shrugging:

Reply:
I need to dig into how Probot is actually logging straight to stderr and ignoring the log level

Reply:
:thinking:

Reply:
Probot logging has been an issue since the beginning apparently, Erwin has mentioned it a few times, and it doesn't always log properly to GCE logs either. It's a known issue that just hasn't made it to the top of the priority list, but I imagine it's annoying in the Travis logs so I can bump it up and look at it next week

Reply:
Ok. It's interesting that the other bots that also use Probot don't have the same issue.

Reply:
I know pretty little about Probot, and have mostly been updating the endpoints that were there, so maybe @danielrozenberg is a bit more familiar with how the actual app config works and why this is happening

Reply:
#457